src/endpoints/users.js

```diff
@@ -0,0 +1,77 @@
+import express from 'express'
+import { handleError } from '../error.js'
+import _ from 'lodash'
+
+const usersRouter = express.Router()
+
+const apiBaseUrl = process.env.API_BASE_URL
+const historySize = {
+  resource: Number(process.env.RESOURCE_HISTORY_SIZE) || 10,
+  template: Number(process.env.TEMPLATE_HISTORY_SIZE) || 10,
+  search: Number(process.env.SEARCH_HISTORY_SIZE) || 10,
+}
+
+usersRouter.post('/:userId', (req, res) => {
+  console.log(`Received post to ${req.params.userId}`)
```

Comment:
Do we tend to use `console.log` instead of log-level-specific invocations in sinopia_api, e.g., `console.debug`? Either is fine with me, as long as we're consistent, and I assume that's why you chose `console.log` here.

Reply:
This is consistent with the rest of the API. However, using `console.log` is used without any consideration of Express best practices. https://github.com/LD4P/sinopia_api/issues/62

Reply:
:+1: